services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/volume_removal.py

```diff
@@ -0,0 +1,35 @@
+from servicelib.rabbitmq_utils import RPCNamespace
+
+from ..rabbitmq import RabbitMQClient
+
+
+async def remove_volumes_from_node(
+    rabbitmq_client: RabbitMQClient,
+    volume_names: list[str],
+    docker_node_id: str,
+    *,
+    volume_removal_attempts: int = 15,
```

Comment:
attempts?? how is this defined? why attempts here and not timeouts? This is a bit strange an interface

Reply:
THOUGHT: I am also very skeptical when it comes to so many timing tunings. If possible please keep all these to a minimal and simple interface. Please double check all the timings you have already to adjust in the dy-sidecar and consider how we could get rid of some of them.